src/main/java/dk/kb/datahandler/config/ServiceConfig.java

```diff
@@ -49,6 +49,7 @@ public static synchronized void initialize(String configFile) throws IOException
         oaiTimestampFolder= serviceConfig.getString("config.timestamps.folder");
         dsStorageUrl = serviceConfig.getString("config.storage.url");
         solrUrl = serviceConfig.getString("config.solr.url");
+        solrBatchSize=  serviceConfig.getInteger("config.solr.batchSize");
```

Comment:
For low priority things like this, please add a default value to the `getInteger` call. I'll accept the pull request as one is given in the `ds-datahandler-behaviour.yaml`.